Doc/library/enum.rst

```diff
@@ -424,9 +424,9 @@ Data Types
       Using :class:`auto` with :class:`IntEnum` results in integers of increasing
       value, starting with ``1``.
 
-   .. versionchanged:: 3.11 :meth:`__str__` is now :func:`int.__str__` to
+   .. versionchanged:: 3.11 :meth:`!__str__` is now :meth:`!int.__str__` to
```

Comment:
```suggestion
   .. versionchanged:: 3.11 :meth:`~object.__str__` is now :meth:`!int.__str__` to
```

As discussed, as readers may not necessarily be familiar with exactly what `__str__` is or how it is used in Python, it seems reasonable to have `__str__` in the rendered output linked to the canonical description of the `__str__` dunder.